lib/tasks/custom_seeds.rake

```diff
@@ -8,15 +8,22 @@ namespace :db do
       raise "Attempting to seed production environment, aborting!" if Rails.env.production?
 
       filename = SEED_DIR.join("seeds_e2e.rb")
-      load(filename) if File.exist?(filename)
```

Comment:
We should want this to fail loudly. This seed file is critical to E2E testing and I would like to know when it goes missing.